src/query/frontend/ast/cypher_main_visitor.cpp

```diff
@@ -2204,6 +2207,21 @@ antlrcpp::Any CypherMainVisitor::visitLiteral(MemgraphCypher::LiteralContext *ct
   return visitChildren(ctx);
 }
 
+antlrcpp::Any CypherMainVisitor::visitExistsExpression(MemgraphCypher::ExistsExpressionContext *ctx) {
+  auto *exists = storage_->Create<Exists>();
+  exists->pattern_ = std::any_cast<Pattern *>(ctx->patternPart()->accept(this));
+
+  if (exists->pattern_->identifier_) {
+    throw SyntaxException("No pattern identifiers can be present in exists!");
```

Comment:
```suggestion
    throw SyntaxException("Identifiers are not supported in exists(...).");
```

Reply:
I just raise awarness here without expecting any action: There is a difference between syntax and semantic exceptions. Obviously they are for syntactical or semantical errors. However I am not sure which category do these errors fall into, so that's why I don't know whether they are correct or not.

Therefore the goal of this comment is spreading knowledge and providing information to you so you can make the best decision.

Reply:
I think it's okay that we here make a syntax exception. However, if I make this an anonymous pattern part, there can not be any mistake that the user can not enter an identifier in the exists() part. However, I'd still need to check in the other parts of the code that identifier is not set since it's the same Pattern object which can contain an identifier (done when pathfinding is performed).

Reply:
@antaljanosbenjamin  @antoniofilipovic

Reply:
For me this is okay. I will resolve the comment.